### DISCUSSION TITLE
docusaurus-comments

### DISCUSSION THREAD

**Discussion by giscus** [AMD-NICK/blog.amd-nick.me#11] [Blog comments]
# docusaurus-comments

gisqus

https://blog.amd-nick.me/docusaurus-comments

**Comment by AMD-NICK** [comment]
# 👋 Hello world!

**Comment by shockpast** [comment]
Много времени искал название базы твоего блога, и прочитав буквально первый абзац, сразу нашёл ответ, сама фича с комментариями выглядит интересно :)

**Reply by AMD-NICK** [reply]
Когда-то давненько искал как сделать комменты на Disqus и каждый раз откладывал это из-за того, что оно как-то костыльно все казалось. Но вот сделал и вроде норм солюшин. Довольно просто оказалось и не так уж костыльно
